pageserver/src/tenant/tasks.rs

```diff
@@ -192,20 +192,28 @@ async fn compaction_loop(tenant: Arc<Tenant>, cancel: CancellationToken) {
                 }
             }
 
-            let started_at = Instant::now();
 
-            let sleep_duration = if period == Duration::ZERO {
+
+            let sleep_duration;
```

Comment:
why make this refactoring, what's the benefit?

Reply:
The `else` branch starting in line 203 would otherwise require an additional temporary local variable, so that we can store the `match`-dependent `sleep_duration`, do the `tracing::debug!` in line 237, then put the temporary variable as an expression as the last line inside `else`.

Don't feel strongly, can revert if it makes you happy

Reply:
Aah now I see, it's fine.